config/src/main/java/com/alibaba/fescar/config/ConfigType.java

```
@@ -45,14 +48,11 @@
      * @return the type
      */
     public static ConfigType getType(String name) {
-        if (File.name().equalsIgnoreCase(name)) {
-            return File;
-        } else if (Nacos.name().equalsIgnoreCase(name)) {
-            return Nacos;
-        } else if (Apollo.name().equalsIgnoreCase(name)) {
-            return Apollo;
-        }else {
-            throw new NotSupportYetException("unsupport type:" + name);
+        for (ConfigType configType : values()) {
+            if (configType.name().equalsIgnoreCase(name)) {
+                return configType;
+            }
         }
+        return Default;
```

Let's use `throw Illegal Argument` instead of `return Default` here.